### ISSUE TITLE
Issue with last version of `inputmask`

### ISSUE THREAD

**Issue by alex-konoval** [brandynbennett/ember-inputmask#87]
Hi guys, there is an issue with last version of `inputmask`. They changed path to `inputmask.phone.extensions` and now it's impossible to run it, get an error:
```
ENOENT: no such file or directory, open '/home/circleci/app/tmp/source_map_concat-input_base_path-eyDKFPNJ.tmp/vendor/inputmask/dist/inputmask/inputmask.phone.extensions.js'
```
To solve it path should be fixed for `0.8` and `0.7`.
Thanks.

**Comment by minusfive** [comment]
Yup, been hitting the same error.

**Comment by brandynbennett** [comment]
Fixed in 0.8.1

**Comment by alex-konoval** [comment]
@brandynbennett Thanks for fix. But could you please do the same for `0.7.2` (`0.7.3`) for users who use `Ember 2.17.0`.

**Comment by brandynbennett** [comment]
@alex-konoval I pushed up the fix in `0.7.3`. I'm curious to know if there are compatibility issues with upgrading to 0.8.x? I have tests running back to ember@2.16 which have been passing. Did I miss something??

**Comment by alex-konoval** [comment]
Thank you.
I guess it will fetch extra version of Ember package. But, maybe I'm wrong with this assumption.
